index.html

```diff
@@ -19,21 +20,43 @@ <h5>Average Grade <span class="badge badge-secondary"></span></h5>
       </div>
     </div>
   </header>
-  <table class="table table-striped container">
-    <thead>
-      <tr>
-        <th scope="col">Student Name</th>
-        <th scope="col">Course</th>
-        <th scope="col">Grade</th>
-      </tr>
-    </thead>
-    <tbody>
-    </tbody>
-  </table>
+  <div class="container">
+    <div class="row">
+      <div class="col-sm-8">
+        <table class="table table-striped">
+          <thead>
+            <tr>
+              <th scope="col">Student Name</th>
+              <th scope="col">Course</th>
+              <th scope="col">Grade</th>
+            </tr>
+          </thead>
+          <tbody>
+          </tbody>
+        </table>
+      </div>
+      <div class="col-sm-4">
+        <form>
+          <div class="form-group">
+            <input type="text" class="form-control" name="name" placeholder="Name">
```

Comment:
Did you forget to do the font awesome icons?

Reply:
yes haha